modules/virtualization/docker.nix

```diff
@@ -1,27 +0,0 @@
-# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
-# SPDX-License-Identifier: Apache-2.0
-{
-  config,
-  pkgs,
-  modulesPath,
-  ...
-}: let
-  cfg = config.ghaf.virtualization.docker.daemon;
-in
-  with lib; {
-    options.ghaf.virtualization.docker.daemon = {
```

Comment:
Did you accidentally remove this module in this PR?

Although I think it is not very useful module, basically one can just set the virtualisation.docker options directly if one wants

Reply:
No, it was on purpose. The module didn't work and caused infinite recursion during evaluation because of conditional import in the config attribute.

Reply:
Although I think it is a quite pointless module, I'd still like to explore the route of trying to fix it first in another PR, instead of completely removing it in this PR.

Reply:
Fixed in [https://github.com/tiiuae/ghaf/pull/217](https://github.com/tiiuae/ghaf/pull/217/files)